spec/views/content/edit.html.erb_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+RSpec.describe "content/edit.html.erb" do
+  describe "edit document form" do
+    it "should render correctly for every document type" do
+      document_types_we_manage = DocumentType.all.filter { |dt| dt.managed_elsewhere.blank? }
+      document_types_we_manage.each do |document_type|
```

Comment:
I wonder if this can be covered implicitly by feature tests for each format ensuring we hit the edit view? I'm guessing for coverage we need each to visit each supported documents edit content page, or is it more complex than that?

Reply:
I think on balance feature tests are a better idea. We had one already, but removed it here: https://github.com/alphagov/content-publisher/pull/1710/commits/66e153d8a0bd39831266e91132fffa66bf7afdf7

I could look at reintroducing it, but without the repetition between press_release and news_story.

Reply:
Spoke to Ben IRL about this, and decided as "rendering an edit form" isn't a self-contained feature, then we should not really write a feature test for it (especially as we're adding many more formats soon!)

Hopefully have now covered this satisfactorily in the schema test.